zebra-network/src/meta_addr.rs

```diff
@@ -524,9 +528,8 @@ impl MetaAddr {
 
         Some(MetaAddr {
             addr: canonical_socket_addr(self.addr),
-            // TODO: split untrusted and direct services
-            //       sanitize untrusted services to NODE_NETWORK only? (#2234)
-            services: self.services,
+            // initial peers are sanitized assuming they are `NODE_NETWORK`
+            services: self.services.or(Some(PeerServices::NODE_NETWORK)),
```

Comment:
This change deletes some TODOs.

But the sanitization and field split aren't actually implemented in this PR.
So let's fix the ticket reference to #2324 and keep this TODO comment.

```suggestion
            // initial peers are sanitized assuming they are `NODE_NETWORK`
            // TODO: split untrusted and direct services
            //       consider sanitizing untrusted services to NODE_NETWORK (#2324)
            services: self.services.or(Some(PeerServices::NODE_NETWORK)),
```